### ISSUE TITLE
Can Kanban be arranged in multiple rows?

### ISSUE THREAD

**Issue by zhl111** [mgmeyers/obsidian-kanban#193]
![image](https://user-images.githubusercontent.com/29135984/121761558-16398000-cb63-11eb-8a91-0a226a4ca374.png)

**Comment by zhl111** [comment]
![image](https://user-images.githubusercontent.com/29135984/121834147-98ab7680-cd00-11eb-86b5-61a643c199de.png)
![Uploading image.png…]()

**Comment by roaldarbol** [comment]
I second this feature request; I find it useful to have a board with tasks divided by project, and then my prioritised to-do list that I can copy the tasks onto. This can of course be achieved by having 2 boards in the same view with a horizontal split, only this have to be manually toggled every time the view is required, making it unfeasible.
Lastly, just a big thanks for making this amazing plug-in! I switched to Obsidian a few months ago, but then felt the need to also use Trello on the side - so I'm super excited that this plugin allows me to keep all of my workflow within Obsidian!

**Comment by tlnagy** [comment]
This seems related to the concept of [swimlanes](https://kanbantool.com/support/kanban-board/what-are-swimlanes) as brought up in #237.

**Comment by alexisthual** [comment]
Thank you for the great plugin! Swimlanes would be a total game-changer, I don't think I would ever need a different task managing tool if this is implemented :smile:

**Comment by alexisthual** [comment]
Here are some thoughts on how it could be achieved:
- each row would be associated with a unique tag, eg `#project1`, `#project2`, etc, with the first row having no specific tag
- this list of tags would be part of the kanban settings appended to the markdown file
- if a given card belongs to several rows, display it only once (need to discuss criteria, but this should be straightforward)

I see several advantages to this setting:
- it minimises changes made to the global structure of markdown files
- it ensures that all rows have the same columns
- at first sight, I shouldn't be that hard to implement?

However, I am not sure moving a card between rows would work right away (I didn't give a close look at the dnd part).
I am willing to work on this if you think this approach sounds okay @mgmeyers

**Comment by g-berthiaume** [comment]
+1 for swimlanes. 
IMO, It would greatly improve ergonomics, while not being too intrusive on the markdown.

**Comment by mreel** [comment]
> IMO, It would greatly improve ergonomics, while not being too intrusive on the markdown schema.

Exactly, columns are already second level headers, so top level headers could be used for the swimlanes!

**Comment by chromer030** [comment]
+1 for swimlanes.
Please plan it as high priority , it can be game changer for many of us.

Thanks alot.

**Comment by digitalsignalperson** [comment]
Here's a css snippet to wrap cards to multiple rows.

```css
.kanban-plugin__board > div {
    display: flex;
    flex-wrap: wrap;
    row-gap: 40px;
}
```

**Comment by beelzebubbles** [comment]
+1 for swimlanes.

**Comment by codescv** [comment]
> Here's a css snippet to wrap cards to multiple rows.
> 
> ```css
> .kanban-plugin__board > div {
>     display: flex;
>     flex-wrap: wrap;
>     row-gap: 40px;
> }
> ```

Thanks!  but when I use this script the scrolling stopped working

**Comment by jonasgutzke** [comment]
> > Here's a css snippet to wrap cards to multiple rows.
> > ```css
> > .kanban-plugin__board > div {
> >     display: flex;
> >     flex-wrap: wrap;
> >     row-gap: 40px;
> > }
> > ```
> 
> Thanks! but when I use this script the scrolling stopped working

Add `overflow-y:` auto; that fixed it for me.
